receiver/pcapreceiver/README.md

```diff
@@ -55,20 +55,11 @@ tcpdump --version
 
 ### Windows
 
-**Tool**: Requires Wireshark (which includes Npcap). Ensure `dumpcap.exe` is available (typically from Wireshark installation, or specify `executable_path`).
+**Tool**: Requires Npcap driver (included with Wireshark, or install standalone from https://npcap.com/).
```

Comment:
Can you enlighten to me if we still need the Npcap driver? OR is the golang library sufficient? I feel a tad bit confused with the language on the dependencies of the new library.